components/FlipTracking/FlipTracking.tsx

```diff
@@ -143,14 +143,47 @@ export function FlipTracking(props: Props) {
                     </div>
-                    <div style={{ display: 'flex', alignItems: 'end' }}>
-                        <CopyButton
-                            copyValue={
-                                isClientSideRendering() ? `${window.location.origin}/player/${router.query.uuid}/flips/${trackedFlip.uId.toString(16)}` : ''
-                            }
-                            successMessage={isClientSideRendering() ? <span>{`Copied link to flip!`}</span> : <span />}
-                        />
+                </div>
+                <p
+                    style={{ cursor: 'pointer' }}
+                    onClick={() => {
+                        let flips = [...trackedFlips]
+                        trackedFlip.showPropertyChanges = !trackedFlip.showPropertyChanges
+                        setTrackedFlips(flips)
+                    }}
+                >
+                    Extra profit calculations: {trackedFlip.showPropertyChanges ? <ArrowDownIcon /> : <ArrowRightIcon />}
+                </p>
+                {trackedFlip.showPropertyChanges ? (
+                    <div style={{ display: 'flex', justifyContent: 'space-between' }}>
+                        <Table>
+                            <tbody>
+                                {trackedFlip.propertyChanges.map(change => (
+                                    <tr>
+                                        <td>{change.description}</td>
+                                        <td>
+                                            {' '}
+                                            {change.effect > 0 ? (
+                                                <span style={{ color: 'lime', whiteSpace: 'nowrap', marginLeft: '5px' }}>
+                                                    +{numberWithThousandsSeperators(change.effect)} Coins
+                                                </span>
+                                            ) : (
+                                                <span style={{ color: 'red', whiteSpace: 'nowrap', marginLeft: '5px' }}>
+                                                    {numberWithThousandsSeperators(change.effect)} Coins
+                                                </span>
+                                            )}
+                                        </td>
+                                    </tr>
+                                ))}
+                            </tbody>
+                        </Table>
                     </div>
+                ) : null}
+                <div style={{ display: 'flex', justifyContent: 'end' }}>
+                    <CopyButton
+                        copyValue={isClientSideRendering() ? `${window.location.origin}/player/${router.query.uuid}/flips/${trackedFlip.uId.toString(16)}` : ''}
+                        successMessage={isClientSideRendering() ? <span>{`Copied link to flip!`}</span> : <span />}
+                    />
```

Comment:
could the copy button be on the same height as the text if the profit changes are not extended like it was before?  
Currently the button takes an extra line.  
![image](https://user-images.githubusercontent.com/16632490/193137270-7dfcd9ab-5e5b-4715-96f2-09f389726d38.png)

Also could you inline the sell time with the finder, and the cherry on top would be if you displayed the relative time like on the auction paage :)